demo/summer-2016.html

```diff
@@ -288,12 +288,10 @@
       }
       $("#highlightInfo").popover("destroy");
     };
-
     //Initialization bind for highlight brackets.
     highlighter.init(ace, editor, {win: 'Ctrl-Shift-9', mac: 'Command-Shift-9'}, candidateToggleSurroundingBracketsPopup);
-
-    langTools.addCompleter(texCompleter)
-    texCompleter.init(editor, {win: 'Enter', mac: 'Enter'})
+    completionTools.init(editor,  {win: 'enter', mac: 'enter'})
```

Comment:
it looks strange that module initialization goes after using some code from the module (at line 239). Besides, langTools is accessed both here and at line 240. Move lines 239-240 below this init?

Also, I suggest creating a static method TexCompleter.init (you need to prefix method definition with `@` in CS) and returning just TexCompleter from the module instead of using `exports` (we do it in Papeeria code anyway). So that here you could write
```
require([...], -> (... TexCompleter...)
  TexCompleter.init()
  langTools.setCompleters([new TexCompleter()])
```